pygeo/mphys/mphys_dvgeo.py

```diff
@@ -125,7 +128,21 @@ def compute(self, inputs, outputs):
         # next time the jacvec product routine is called
         self.update_jac = True
 
-    def nom_addChild(self, ffd_file, DVGeoName=None, childName=None):
+    def nom_addComponent(self, comp, ffd_file, triMesh, DVGeoName=None):
```

Comment:
Can you add a docstring for this function since you added it??

Reply:
Added.